test/functional/http/get/transactions.js

```diff
@@ -592,169 +602,71 @@ describe('GET /api/transactions', function () {
-				var sort = 'amount:asc';
-
-				var params = [
-					'senderId=' + accountFixtures.genesis.address,
-					'recipientId=' + account.address,
-					'recipientId=' + account2.address,
-					'limit=' + limit,
-					'offset=' + offset,
-					'sort=' + sort
-				];
-
-				return getTransactionsPromise(params).then(function (res) {
-					expect(res).to.have.property('status').to.equal(200);
-					expect(res).to.have.nested.property('body.transactions').that.is.an('array');
-					expect(res).to.have.nested.property('body.transactions').that.have.length.within(transactionList.length, limit);
-					for (var i = 0; i < res.body.transactions.length; i++) {
-						if (res.body.transactions[i + 1]) {
-							expect(res.body.transactions[i].amount).to.be.at.most(res.body.transactions[i + 1].amount);
-						}
-					}
-				});
-			});
 
-			it('using many valid parameters should be ok', function () {
-				var limit = 10;
-				var offset = 0;
-				var sort = 'amount:asc';
+				return transactionsEndpoint.makeRequest({
+					senderAddress: accountFixtures.genesis.address,
+					recipientAddress: account.address,
+					limit: 10,
+					offset: 0,
+					sort: 'amount:asc'
+				}, 200).then(function (res) {
 
-				var params = [
-					'blockId=' + '1',
-					'senderId=' + accountFixtures.genesis.address,
-					'recipientId=' + account.address,
-					'fromHeight=' + 1,
-					'toHeight=' + 666,
-					'fromTimestamp=' + 0,
-					'minAmount=' + 0,
-					'limit=' + limit,
-					'offset=' + offset,
-					'sort=' + sort
-				];
+					var values = _.map(res.body.data, 'amount').map(function (value) { return parseInt(value); });
+					_(_.clone(values)).sortNumbers('asc').should.be.eql(values);
 
-				return getTransactionsPromise(params).then(function (res) {
-					expect(res).to.have.property('status').to.equal(200);
-					expect(res).to.have.nested.property('body.transactions');
+					res.body.data.forEach(function (transaction) {
+						transaction.senderAddress.should.be.eql(accountFixtures.genesis.address);
+						transaction.recipientAddress.should.be.eql(account.address);
+					});
 				});
 			});
 		});
 
-		describe('count', function () {
+		describe('meta', function () {
 
-			it('should return count of the transactions with response', function () {
-				return getTransactionsPromise({}).then(function (res) {
-					expect(res).to.have.property('status').to.equal(200);
-					expect(res).to.have.nested.property('body.transactions').that.is.an('array');
-					expect(res).to.have.nested.property('body.count').that.is.a('string');
+			describe('count', function () {
+				it('should return count of the transactions with response', function () {
+					return transactionsEndpoint.makeRequest({}, 200).then(function (res) {
+						res.body.meta.count.should.be.a('number');
+					});
 				});
 			});
 		});
 	});
 
-	describe('/count', function () {
+	// TODO: After enabling new paths in swagger, fix these tests.
+	describe.skip('/count', function () {
```

Comment:
@karmacoma These test cases will be enabled when we migrate other endpoints in swagger.

Reply:
Ok